runtime/ztests/expr/from-eval-subquery.yaml

```diff
@@ -0,0 +1,22 @@
+script: super -s -I query.spq
```

Comment:
Nit: Call this querexpr-from-eval.yaml so it's grouped with the other queryepxr-* tests.